identityClient/api.go

```diff
@@ -1269,3 +1269,64 @@ type UpdatePAMJiraPluginRequest struct {
 
 // UpdatePAMJiraPluginResponse is the updated PAM Jira plugin information
 type UpdatePAMJiraPluginResponse = PAMJiraPlugin
+
+// InitiateWebAuthnChallengeRequest wraps info for initiating the registration process of a WebAuthn device
+type InitiateWebAuthnChallengeRequest struct {
+	IdentityID   string `json:"identity_id"`
+	SessionToken string
+}
+
+// InitiateWebAuthnChallengeResponse wraps the challenge login action data for WebAuthn
+// This is the login action that comes directly from keycloak.
+type InitiateWebAuthnChallengeResponse struct {
+	TabID         string                           `json:"tab_id"`
```

Comment:
For each of these fields, think about the time it took you to know what each was for, and then consider whether you and future developers would appreciate some context revealing comments. Adding code comments is one approach, a confluence page and some lucid charts are another approach, as long as we're documenting the context 👍🏽